Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```diff
@@ -890,6 +890,11 @@
         cos: "the cosine of %s",
       },
 
+      invalid_operator_args: {
+        single: "Got an unexpected iota at index %d of the stack: %s",
+        plural: "Got %d unexpected iotas at indices %d-%d of the stack: %s",
```

Comment:
nit: I think `one` and `many` would be more similar to other existing keys; not by any means a huge issue though, up to you